app/scripts/models/auth_brokers/oauth.js

```diff
@@ -63,20 +62,19 @@ function (_, Constants, Url, OAuthErrors, AuthErrors, p, Validate,
 
     getOAuthResult: function (account) {
       var self = this;
-      if (! account || ! account.get('sessionToken')) {
-        return p.reject(AuthErrors.toError('INVALID_TOKEN'));
-      }
-
-      return self._assertionLibrary.generate(account.get('sessionToken'))
-        .then(function (assertion) {
+      // Ensure we always return a promise, even on error.
+      return p()
+        .then(function() {
```

Comment:
Running "jscs:app" (jscs) task
One space required after "function" keyword at app/scripts/models/auth_brokers/oauth.js :
    65 |      // Ensure we always return a promise, even on error.
    66 |      return p()
    67 |        .then(function() {
------------------------------^
    68 |          if (! account || ! account.get('sessionToken')) {
    69 |            return p.reject(AuthErrors.toError('INVALID_TOKEN'));
>> 1 code style errors found!
Warning: Task "jscs:app" failed. Use --force to continue.
Aborted due to warnings.